src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -264,6 +264,27 @@ FabricInfo * RetrieveCurrentFabric(CommandHandler * aCommandHandler)
     return Server::GetInstance().GetFabricTable().FindFabricWithIndex(index);
 }
 
+CHIP_ERROR DeleteFabricFromTable(FabricIndex fabricIndex)
+{
+    ReturnLogErrorOnFailure(Server::GetInstance().GetFabricTable().Delete(fabricIndex));
```

Comment:
Please do not use `ReturnLogErrorOnFailure` in server code.  It's very bloaty due to use `__FILE__`.
```suggestion
    ReturnErrorOnFailure(Server::GetInstance().GetFabricTable().Delete(fabricIndex));
```